RetailCoder.VBE/Settings/GeneralSettings.cs

```diff
@@ -8,29 +8,29 @@ namespace Rubberduck.Settings
     public interface IGeneralSettings 
     {
         DisplayLanguageSetting Language { get; set; }
-        bool ShowSplash { get; set; }
-        bool CheckVersion { get; set; }
-        bool SmartIndenterPrompted { get; set; }
-        bool AutoSaveEnabled { get; set; }
+        bool CanShowSplash { get; set; }
+        bool CanCheckVersion { get; set; }
+        bool IsSmartIndenterPrompted { get; set; }
+        bool IsAutoSaveEnabled { get; set; }
```

Comment:
Why change the names of our settings? In particular `CanShowSplash` is misleading.

Reply:
I had seen `bool` with `Is...` and others without. The thought of standardizing them a bit came to mind. https://chat.stackexchange.com/transcript/14929?m=41324402#41324402 led me to add them. Is this something I should revert?

Reply:
Yes, please. That said IIRC the autosave feature is long gone, ..would removing that setting cause issues?

Reply:
if it does, then those issues need to be addressed sooner or later anyways. So I'm all for removing it ...